app/src/main/AndroidManifest.xml

```diff
@@ -185,6 +185,8 @@
     <activity android:name=".app.testing.ImageRegionSelectionTestActivity" />
     <activity android:name=".app.testing.ImageViewBindingAdaptersTestActivity" />
     <activity android:name=".app.testing.MarginBindingAdaptersTestActivity" />
+    <activity android:name=".app.testing.DrawableBindingAdaptersTestActivity"
```

Comment:
Please keep these activities in lexicographical order by their android:name property.

Reply:
Done.